src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/xlf/Strings.cs.xlf

```diff
@@ -152,6 +152,11 @@
         <target state="translated">Třída InternalTypeHelper není pro tento projekt požadována, vyprázdněte soubor {0}.</target>
         <note />
       </trans-unit>
+      <trans-unit id="InvalidAssemblyVersion">
```

Comment:
We need to notify the localizers about this.

Reply:
The localization team has automation that should detect and update based on these changes (I have an email to that affect:

```
Currently all 3 locations are monitored daily by automation and we keep weekly cycles of localization. In case of new string is added we deliver PR to repository.
```

In any case, I will ping them as a heads up once we merge.